### PR TITLE
Feature: Added shortcut to open Windows Terminal as administrator 

### LINKED ISSUE #11278: Feature: Open Windows Terminal as admin using ctrl + shift + `
### What feature or improvement do you think would benefit Files?

On Windows 10, on MS File Explorer (MSFE) you can open a shell terminal by using the combo ALT->F->S->(A|R) A for Admin or R for Regular User.  
This is **extremely** handy as one can open a shell as admin on the path we want to work with with ease, especially for software installation etc.  

image

On Windows 11, the brilliant minds of Microsoft removed any of the ALT keys from MSFE, because reasons.  

I've started using Files, I like the way it works, and having this feature would be great. 

### Requirements

- This proposal will require the addition of a hot-key or sequence of keys that would open a (power)shell terminal on the current location as administrator. 

### Files Version

2.4.33.0

### Windows Version

Win 10 - 10.0.19044.2486

### Comments

_No response_

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11278 

**Notes**
- Fixes a bug with #11192 which would not open the path if the default terminal was PowerShell

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Review by yaira2 — APPROVED** [review]
LGTM

**Comment by yaira2** [comment]
Instead of requiring Files to be admin, can display a UAC prompt?

**Comment by ferrariofilippo** [comment]
There's a bug with latest commit, I'll try to figure out why

**Review by yaira2 — APPROVED** [review]
LGTM
Can you add to the documentation?

File: src/Files.App/Views/ColumnShellPage.xaml.cs [hunk 1]
```diff
@@ -230,7 +230,7 @@ private async void ColumnShellPage_PreviewKeyDown(object sender, KeyRoutedEventA
 
 			switch (c: ctrl, s: shift, a: alt, t: tabInstance, k: args.Key)
 			{
-				case (true, false, false, true, (VirtualKey)192): // ctrl + ` (accent key), open terminal
+				case (true, _, false, true, (VirtualKey)192): // ctrl + ` (accent key), open terminal
```
Comment (francisco-lafe) [inline src/Files.App/Views/ColumnShellPage.xaml.cs:233]: Shouldn't it say "true" instead of "_"?

Comment (francisco-lafe) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: I think the way this should be done is with another case where Shift is 'true' and then there pass the "-Verb runas" as an argument.

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: I don't think so, it would only create duplicate code

Comment (francisco-lafe) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: Alright, I don't know if a following commit trampled over these changes, but the feature for "CTRL+SHIFT+`" is not working. 
#11443

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: I'll check

Comment (francisco-lafe) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: > I'll check

I did, but unsure if the feature was moved elsewhere. On commit:
https://github.com/files-community/Files/commit/dc676d30a6e01836683e9bde4bed88ac92f65be5

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: Moved elsewhere but still it should work. I'll see I can find what is causing the problem

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: Can you confirm that it was working before that commit?

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: Ok I found the root and I feel so stupid ...

Comment (francisco-lafe) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: > Ok I found the root and I feel so stupid ...

You're smart enough to have found it! 
Great work 😄

Comment (ferrariofilippo) [reply src/Files.App/Views/ColumnShellPage.xaml.cs:233]: I wouldn't have without your help

**Pull request merged**